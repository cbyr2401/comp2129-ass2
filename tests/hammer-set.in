LIST ENTRIES
SET a 1 2 3
LIST ENTRIES
SET b 4 5 6 7 8 9
LIST ENTRIES
SET c 10
LIST ENTRIES
SET fibonacci 1 1 2 3 5 8 13 21 34 55 89 144
LIST ENTRIES
SET catalan 1 2 5 14 42 132 429 1430 4862 16796
LIST ENTRIES
LIST KEYS
LIST SNAPSHOTS
DEL c
LIST KEYS
LIST SNAPSHOTS
LIST ENTRIES
PURGE e
LIST ENTRIES
PURGE a
LIST ENTRIES
BYE