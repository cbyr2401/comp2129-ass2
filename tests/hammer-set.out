> no entries

> ok

> a [1 2 3]

> ok

> b [4 5 6 7 8 9]
a [1 2 3]

> ok

> c [10]
b [4 5 6 7 8 9]
a [1 2 3]

> ok

> fibonacci [1 1 2 3 5 8 13 21 34 55 89 144]
c [10]
b [4 5 6 7 8 9]
a [1 2 3]

> ok

> catalan [1 2 5 14 42 132 429 1430 4862 16796]
fibonacci [1 1 2 3 5 8 13 21 34 55 89 144]
c [10]
b [4 5 6 7 8 9]
a [1 2 3]

> catalan
fibonacci
c
b
a

> no snapshots

> ok

> catalan
fibonacci
b
a

> no snapshots

> catalan [1 2 5 14 42 132 429 1430 4862 16796]
fibonacci [1 1 2 3 5 8 13 21 34 55 89 144]
b [4 5 6 7 8 9]
a [1 2 3]

> ok

> catalan [1 2 5 14 42 132 429 1430 4862 16796]
fibonacci [1 1 2 3 5 8 13 21 34 55 89 144]
b [4 5 6 7 8 9]
a [1 2 3]

> ok

> catalan [1 2 5 14 42 132 429 1430 4862 16796]
fibonacci [1 1 2 3 5 8 13 21 34 55 89 144]
b [4 5 6 7 8 9]

> bye
